lamps-rfc7030-csrattrs.mkd

```diff
@@ -195,7 +212,7 @@ with no change to the bits on the wire.
 
 ## Use of CSR templates {#csrtemplate}
 
-Alternatively to the piecemeal inclusion of CSR attributes
+Alternatively to the partial inclusion of CSR attributes
```

Comment:
The main problem I see with the RFC7030 CSR attributes approach is not partiality (which often is needed),
but that it is amorph: there is no structure in the list of attributes, which makes the requirements hard to interpret and can lead to ambiguities.
So please replace `partial` by, e.g., `unstructured`.